lib/servers_testnet.json

```diff
@@ -5,11 +5,18 @@
     "blackie.c3-soft.com": {
         "s": "60002"
     },
+    "cash-testnet.theblains.org": {
+        "s" : "50004"
```

Comment:
You need commas in between items in a map (dict) in JSON.  Comma missing here is still an error...